examples/khr-3hv/khr-3hv_continuous/controllers/supervisorsManager/khr3hvEnv.py

```diff
@@ -0,0 +1,206 @@
+            3	RightAnkle  -2.35    +2.35    Set the motor position from -2.35 to +2.35 
+            4	RightCrus   -2.35    +2.35    Set the motor position from -2.35 to +2.35 
+            5	RightFemur  -2.35    +2.35    Set the motor position from -2.35 to +2.35 
+
+        Reward: 
+            bonus for moving forward 
+            difference of current and pre vious positions
+
+        Starting State:
+            [0, 0, ..., 0]
+
+        Episode Termination:
+            Robot z axis smaller than 0.50 cm
+            Robot walked more that 15 m
+        """
+
+        super().__init__()
+        # Time window of 5
+        self.t = 5
+        # Number of observations
+        self.numObs = 10
+        # Total observations
+        self.obs = np.zeros(self.t * self.numObs)
+        # Lower and maximum values on observation space
+        lowObs = -np.inf * np.ones(self.t * self.numObs)
+        maxObs = np.inf * np.ones(self.t * self.numObs)
+        self.observation_space = Box(low=lowObs, high=maxObs, dtype=np.float64)
+        # Lower and maximum values on action space
+        lowAct = -2.356 * np.ones(6)
+        maxAct = 2.356 * np.ones(6)
+        self.action_space = Box(low=lowAct, high=maxAct, dtype=np.float64)
+
+        # Set up various robot components
+        self.motor_velocity = 2
+        self.robot = self.getSelf()
+        self.setup_agent()
+        self.motorPositionArr = np.zeros(6)
+        self.episodeScore = 0  # Score accumulated during an episode
+        self.prev_pos = self.robot.getPosition()[0]
+
+        # Logging parameters
+        self.distance_walk = 0
+        self.save_freq = wandbSaveFreq
+        self.counterLoging = 0
+        self.train = train
+        self.projectName = projectName
+        self.useRay = useRay
+
+    def get_observations(self):
+        """
+        Create the observation vector on each time step. 
+        The observations are the robot position (x,z,y), robot velocity, 
+        and the state of each motor.
+        :return: Observation vector
+        :rtype: numpy array
+        """
+        motorPos = self.robot.getPosition()
+        motorPos.append(self.robot.getVelocity()[0])
+
+        self.distance_walk = max(
+            self.distance_walk, self.robot.getPosition()[0])
+        self.counterLoging += 1
+        global GLOBAL_DISTANCE
+        GLOBAL_DISTANCE = max(GLOBAL_DISTANCE, self.robot.getPosition()[0])
+        # Logging on the wandb. When using Ray we don't use the custom logging because Ray has it own
+        # logging and produce compatiblity errors of two instaces of wandb.
+        if self.counterLoging % self.save_freq == 0 and self.train and not self.useRay:
+            #wandb.init(project=self.projectName, reinit=False)
+            wandb.log({"Episode distance walked": self.distance_walk,
+                       "Current x position": self.robot.getPosition()[0],
+                       "Globan distance walekd": GLOBAL_DISTANCE
```

Comment:
This also needs to change with GLOBAL_DISTANCE and typos:
```suggestion
                       "Global distance walked": GLOBAL_DISTANCE
```